examples/non-fungible-token/nft/src/lib.rs

```diff
@@ -100,9 +104,64 @@ impl Contract {
     }
 }
 
-near_contract_standards::impl_non_fungible_token_core!(Contract, tokens);
-near_contract_standards::impl_non_fungible_token_approval!(Contract, tokens);
-near_contract_standards::impl_non_fungible_token_enumeration!(Contract, tokens);
+#[near_bindgen]
+impl NonFungibleTokenCore for Contract {
```

Comment:
Let's add it as a doc-string example to [the relevant documentation page](https://docs.rs/near-contract-standards/latest/near_contract_standards/non_fungible_token/core/index.html) (and similarly for other traits). Let's make standard traits more discoverable overall - from the repo README, from [near-contract-standards docs](https://docs.rs/near-contract-standards/latest/near_contract_standards/index.html), from [near-sdk-rs docs](https://docs.rs/near-sdk/latest/near_sdk/).